gitbook-docs/put.md

```diff
@@ -1,11 +1,12 @@
 # PUT Requests
 
-PUT requests are sent to a server to request a change to a value. For example, a client would use PUT to switch the
-anchor light on or off, change the heading of the autopilot, or set position of the anchor.
+PUT requests are sent to a server to request an action to be taken. For example, a client would use PUT to switch the anchor light on or off, change the heading of the autopilot, or change the current input on a stereo.
```

Comment:
Would something like this be a useful addition:

_An action as opposed to reporting a new value is characterised by the possibility of failure: for example the autopilot may not accept the new heading or we may simply fail to communicate with the equipment in question and the request will fail with a timeout._